cocos/dragon-bones/CCSlot.ts

```diff
@@ -87,7 +88,11 @@ export class CCSlot extends Slot {
         this._color = new Color();
     }
 
-    // return dragonBones.CCTexture2D
+    /**
+     * @en Get a Texture2D object from _textureData.
+     * @zh 获取2D纹理对象。
+     * @returns @en If _textureData is null, will return null.  @zh 若 _textureData 为 null，将返回 null。
```

Comment:
If _textureData is null, will return null -> A texture instance if _textureData is not null, otherwize return null.